spec/nosto.load.spec.tsx

```diff
@@ -38,4 +39,24 @@ describe("Nosto client script loading", () => {
     expect(document.querySelector("[nosto-client-script]")?.getAttribute("nosto-language")).toBe("en")
     expect(document.querySelector("[nosto-client-script]")?.getAttribute("nosto-market-id")).toBe("123")
   })
+
+  it("should set loaded state to true when client is loaded", () => {
```

Comment:
should we move the hook execution tests to a different test file?
e.g. useLoadClientScript.spec.ts ?